WordPress/wporg-main-2022
# Replace community image with compressed version

Fixes #97 by replacing the community image with an image compressed with quality `50`, alpha quality `100`, maximum compression encoding time:

```
cwebp community-photo-2.webp -q 50 -alpha_q 100 -m 6 -o community-photo-2-q50.webp
```

This reduces the file size from ~738k to ~263k 🎉 

Props @tellyworth 

Final image: 

![community-photo-2-q50](https://wordpress.org/files/2022/08/community-photo-2-q50.webp)

### How to test the changes in this Pull Request:

1. Load the front page
2. Compare the community image with that on wordpress.org at various sizes
3. Observe minimal degradation
4. Observe that the image blends seamlessly into the background of the block below
